src/js/actions/libraries.js

```diff
@@ -620,9 +621,15 @@ define(function (require, exports) {
             .bind(this)
             .then(function (path) {
                 var layerRef = layerEffectAdapter.referenceBy.current,
-                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path);
-
-                return descriptor.playObject(placeObj);
+                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path),
```

Comment:
Let's change this to playSimpleLayerActions call, so locked layers can be overridden. Right now we throw an error with this pure call if the layer is locked.